book/src/json-output.md

```diff
@@ -0,0 +1,157 @@
+# JSON output
+
+> Structured logging for `probe-run`.
+
+As an alternative to it's human-focused output, `probe-run` offers structured json output. There are two use-cases:
```

Comment:
Its not it's. *it's* == *it is*, while *its* is the gender neutral version of *his* or *hers*.